### PR TITLE
Release notes for 22.xx

### PR DESCRIPTION
**Pull request by neilweinstock**
I'll keep the release notes updated in this PR until release time.

### CODE REVIEW

File: core/ReleaseNotes.md [hunk 1]
```diff
@@ -1,6 +1,83 @@
 Release Notes
 =============
 
+OpenRocket 22.xx Beta 1
```
Comment (wolsen) [inline core/ReleaseNotes.md:4]: note, we'll need to make sure that the [build version](https://github.com/openrocket/openrocket/blob/unstable/core/resources/build.properties#L3) is updated appropriately to ensure that these are consistent.

File: core/ReleaseNotes.md [hunk 2]
```diff
@@ -1,6 +1,83 @@
...
+* Updated motor list to latest data from Thrustcurve.org
+* Added additional fields to motors
+* Disable fin-thickness warnings on phantom tubes
+* Added warning message for phantom body tubes
+* Updated print dialog to allow simulation control
+* Improved Rocksim import
+* Translations:
+  * Added missing Spanish translation
+  * Added Dutch translation
+  * Refined simplified Chinese translation
```
Comment (wolsen) [inline core/ReleaseNotes.md:76]: This is great! I also suspect there have been changes to the english text strings that have not propagated to the final translations. We'll need to iron out how we go about this for final release.

File: core/ReleaseNotes.md (hunk 1 above)
Comment (neilweinstock) [reply core/ReleaseNotes.md:4]: Yes.  I don't really know how we're going to track the builds as we go through the beta, or even how often we will be releasing new builds.

Release notes built into the app would be nice, but I'm not about to ask for that now. :)

File: core/ReleaseNotes.md (hunk 2 above)
Comment (neilweinstock) [reply core/ReleaseNotes.md:76]: I confess I know nothing about the details or status of the translations other than that there were PRs that claimed the above.  I should probably combine those lines into one (added or improved Spanish, Dutch, and Chinese translations).

**Comment by JoePfeiffer** [comment]
One small thing that will come up -- the release notes have been moved to the toplevel directory.

**Comment by neilweinstock** [comment]
Yup, I will fix that.

File: ReleaseNotes.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ Release Notes
 
 OpenRocket 22.00.beta.01
```
Comment (SiboVG) [inline ReleaseNotes.md:4]: Maybe already change this to 22.02.beta.01

Comment (neilweinstock) [reply ReleaseNotes.md:4]: Party pooper.

Comment (neilweinstock) [reply ReleaseNotes.md:4]: Is that what the build number is going to be?  All the screenshots are for 22.00.beta.01, and I'll kill myself before re-doing all those.

**Pull request merged**